packages/devextreme/testing/tests/DevExpress.viz.core/translator2D.tests.js

```diff
@@ -798,6 +798,15 @@ QUnit.test('Translate. Update translator with business range with empty scale br
     assert.strictEqual(translator.translate(450), 1000);
 });
 
+QUnit.test('Translate. Scale breaks. Translator should handle division by zero (T1222629)', function(assert) {
```

Comment:
lets at least name it something like 'Translator should calculate ratioOfCanvasRange correctly'